site/src/entries/chemical-extract.md

```diff
@@ -8,12 +8,23 @@ TODO intro paragraph and how to begin to add
 
 ## Required Information
 
-- TODO
+- **Nickname:** When you have several Isolates, it may be useful to give them short names to help distinguish them. Please follow your instructor's guidelines on how to name entries meaningfully.
+- **Date:** Date: The date the isolate was first created.
+
 
 ## Optional Information
 
-- TODO
+- **Isolate:** Click the "add Isolate" button to link an isolate to this entry. This field is automatically populated for you if you click "Add Eukaryotic Test" to an existing isolate.
```

Comment:
"Add Chemical Extract" instead of "Add Eu. Test"

Reply:
And there is no "add Isolate" button. Instead they "Search for and click" the Isolate that this was a Chem Extract for